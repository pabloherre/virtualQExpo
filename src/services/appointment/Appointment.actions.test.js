import { addAppointment, findAppointments, findAppointmentsFailed, findAppointmentsSuccess } from './Appointment.actions';

describe('SideMenu actions', () => {
  it('should create an action to find appointments', () => {
    expect(findAppointments()).toMatchSnapshot();
  });

  it('should create an action to find appointments with success', () => {
    expect(findAppointmentsSuccess({ name: 'test' })).toMatchSnapshot();
  });

  it('should create an action to find appointments with fail', () => {
    expect(findAppointmentsFailed({ name: 'test' })).toMatchSnapshot();
  });

  it('should create an action to add an appointment event', () => {
    expect(addAppointment({ name: 'test' })).toMatchSnapshot();
  });
});
