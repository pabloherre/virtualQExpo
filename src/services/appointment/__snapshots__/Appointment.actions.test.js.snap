// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu actions should create an action to add an appointment event 1`] = `
Object {
  "type": "APPOINTMENT_ADD_SUCCESS",
}
`;

exports[`SideMenu actions should create an action to find appointments 1`] = `
Object {
  "type": "APPOINTMENT_FIND_PENDING",
}
`;

exports[`SideMenu actions should create an action to find appointments with fail 1`] = `
Object {
  "type": "APPOINTMENT_FIND_FAILED",
}
`;

exports[`SideMenu actions should create an action to find appointments with success 1`] = `
Object {
  "payload": Object {
    "name": "test",
  },
  "type": "APPOINTMENT_FIND_SUCCESS",
}
`;
